projects/1.18/assets/armor-poser/armorposer/lang/zh_cn.json

```
@@ -0,0 +1,32 @@
+{
+    "_comment": "zh_cn",
+
+    "armorposer.gui.title": "盔甲架",
```

```suggestion
    "armorposer.gui.title": "Armor Poser",
```
mod名不翻译（尽管原文就是armorstand）

---

我是工单的作者，不太会用github，请问我要怎么更正翻译错误重新提交呢？

---

https://github.dev/CFPAOrg/Minecraft-Mod-Language-Package/pull/2012
在线vscode